src/index.ts

```diff
@@ -69,7 +73,12 @@ client.on(Events.InteractionCreate, async (interaction) => {
   }
 });
 
+
+
 // when the client is ready, this callback is run only once
 client.once(Events.ClientReady, (readyClient) => {
   logger.info(`Ready! Logged in as ${readyClient.user.tag}`);
+  authApp.listen(port, () => {
```

Comment:
Don't we need to handle graceful shutdown of bot and server?

Reply:
User base is too small to worry about it honestly. But I'd still like to implement it tbh, good point. I'll make an issue for it.